### PR TITLE
One mobile SSP support

### PR DESCRIPTION
**Pull request by vzhukovsky**

### CODE REVIEW

File: src/bidManager.js [hunk 1]
```diff
@@ -203,13 +183,33 @@ class BidManager {
...
+    if (bidRequestConfig.dcn && placementConfig.pos) {
+      return new NexageGetBidRequest(bidRequestConfig, placementConfig);
+    } else if (bidRequestConfig.network && placementConfig.placement) {
+      return new MarketplaceBidRequest(bidRequestConfig, placementConfig);
+    } if (this.isNexagePostRequest(placementConfig.openRtbParams)) {
+      return new NexagePostBidRequest(bidRequestConfig, placementConfig);
+    }
+  }
+
+  isNexagePostRequest(openRtbParams) {
```
Comment (mpereira96) [inline src/bidManager.js]: You do this much more elegantly than me 👍 . Might copy this to be the same in the AOL Container Tag.

File: src/bidManager.js [hunk 2]
```diff
@@ -203,13 +183,33 @@ class BidManager {
...
-  EU: 'adserver.adtech.de',
-  US: 'adserver.adtechus.com',
-  Asia: 'adserver.adtechjp.com'
-};
+  resolveBidRequest(bidRequestConfig, placementConfig) {
+    if (bidRequestConfig.dcn && placementConfig.pos) {
+      return new NexageGetBidRequest(bidRequestConfig, placementConfig);
+    } else if (bidRequestConfig.network && placementConfig.placement) {
+      return new MarketplaceBidRequest(bidRequestConfig, placementConfig);
+    } if (this.isNexagePostRequest(placementConfig.openRtbParams)) {
```
Comment (marian-r) [inline src/bidManager.js]: Missing else

File: src/helpers/ajax.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+
 /**
  * Module for processing ajax requests
  *
  * @param {string} url requested url
- * @param {string} method request method
  * @param {Function} successCallback success result callback
  */
-export function sendRequest(url, method, successCallback) {
+export function sendRequest(url,successCallback, options = {}) {
```
Comment (marian-r) [inline src/helpers/ajax.js]: Missing space

File: src/helpers/utils.js [hunk 4]
```diff
@@ -26,11 +26,37 @@ let utils = {
...
-   * @param {string} documentProtocol document.location.protocol value
    * @returns {string} protocol type
    */
-  resolveHttpProtocol: (documentProtocol) => {
-    return (documentProtocol === 'https:') ? 'https' : 'http';
+  resolveHttpProtocol: () => {
+    return (document.location.protocol === 'https:') ? 'https' : 'http';
+  },
+
+  /***
```
Comment (marian-r) [inline src/helpers/utils.js]: Really minor issue, but 2 asterisks here are enough for JSDoc.

**Pull request merged**